flink-end-to-end-tests/test-scripts/test_streaming_sql.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+################################################################################
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+source "$(dirname "$0")"/common.sh
+
+TEST_PROGRAM_JAR=$TEST_INFRA_DIR/../../flink-end-to-end-tests/flink-stream-sql-test/target/StreamSQLTestProgram.jar
+
+# copy flink-table jar into lib folder
+cp $FLINK_DIR/opt/flink-table*jar $FLINK_DIR/lib
```

Comment:
Hi, @fhueske I have some small questions here. I can see that there are some jar files in the example folder(```$FLINK_DIR/example```), some jar files in the ```$FLINK_DIR/lib``` folder, and other jar files in the ```$FLINK_DIR/opt``` folder. Why did you copy this jar file from opt  folder to lib folder, instead of putting to ```$FLINK_DIR/example``` ? Thanks.

Reply:
Oh. I think because you start taskmanager.sh, so you have to copy that. Not sure about it.

Reply:
@zhangminglei there is a known issue with a classloader that can not be removed. See the notice in https://ci.apache.org/projects/flink/flink-docs-release-1.4/dev/table/#setup. That's why the optional Table API jar is moved to lib.

Reply:
Thanks @twalthr . I will take a look.